CNV-38594: RN Known issue PVC data upload form does not work (#78390)
openshift/openshift-docs: virt/release_notes/virt-4-16-release-notes.adoc

==== Web console
//CNV-38293: 4.16 - Ronen Sde-Or said this was still an issue

> * When you first deploy an {product-title} cluster, creating VMs from templates or instance types by using the web console, fails if you do not have `cluster-admin` permissions.

Reviewer: ```suggestion
* When you deployed an {product-title} cluster, and then created VMs from templates or instance types by using the web console, the VMs failed if you did not have `cluster-admin` permissions. As a workaround, the cluster administrator must first xref:../../nodes/pods/nodes-pods-configmaps.adoc#nodes-pods-configmap-create-from-console_configmaps[create a config map] to provide users access to use templates and instance types to create VMs. (link:https://issues.redhat.com/browse/CNV-38284[*CNV-38284*])
```

Author: This is a 4.15 release note that has been carried over. I only fixed the link syntax.

** As a workaround, the cluster administrator must first xref:../../nodes/pods/nodes-pods-configmaps.adoc#nodes-pods-configmap-create-from-console_configmaps[create a config map] to enable other users to use templates and instance types to create VMs. (link:https://issues.redhat.com/browse/CNV-38284[*CNV-38284*])

//CNV-38594: 4.16 known issue

> * When creating a persistent volume claim (PVC) in the web console by selecting *With Data upload form*, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])

Reviewer: ```suggestion
* When you created a persistent volume claim (PVC) in the web console by clicking/selecting the *With Data upload form* button/option, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])
```

Author: The *With Data upload form* is an item in a drop-down list, so I went with "select" per our [style guidelines](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#references-to-interface-elements).

Perhaps I should change this to:
```suggestion
* When you select *With Data upload form* from the *Create PersistentVolumeClaim* list in the web console, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])
```
